addons/web/controllers/main.py

```diff
@@ -1906,7 +1838,6 @@ def from_data(self, fields, rows):
 
         return xlsx_writer.value
 
-
```

Comment:
@Gorash as I'm changing the file (and because rde has a linter enabled in his editor) do you agree with use the opportunity to re-indent the many routes ? If we were to strictly apply pep8 we would write:

    @http.route([
        '/web/content',
        '/web/content/<string:xmlid>',
        ...
    ])
    def content_common(...):
        ...

Reply:
> and because rde has a linter enabled in his editor

Almost everyone does, what's worrying is that some do not :shrug: 

> do you agree with use the opportunity to re-indent the many routes

No, you don't do useless diff to correct pep8 error.
You just:
- Correct existing error when modifying a line that has error
- Avoid introducing new error

Not sure what you are trying to do here, but if you don't like reviews, what are you even asking for one?

As I said, this PR seems very promising and what a wonderful job you did, I am/was hyped to dig into it, but you seems very refractory to suggestions that are not going your way, already starting with RCO first remarks about useless diff "Too late, here it is, won't change".

Reply:
Calm down, I applied like 95% of your suggestions.

Reviews are not a place where we say "oui amen" to all the suggested changes, it is a place open to discussion where we work *together* to find the best possible solution. I agree that, most of the time the reviewer points out problems and there is nothing to argue about.
Here you are complaining about code that *I am not the author*. I'm not sure I am rightful to change it without other considerations than pure style, let's do it calmly and ask the original author first. :)

Reply:
There is no point changing those lines, I just screenshotted you a random part of the file so you could see that the linter went :boom: and, as I said, that this line diff should probably be removed.

Here, this is just a line in the middle of nowhere that you removed (for no apparent reason) which then introduce a linter error.

See line 1909.. It has nothing to do with gorash :)
![image](https://user-images.githubusercontent.com/30048408/142628873-0452b963-06f3-4abd-8a22-968b820c43be.png)

Maybe you misunderstood my comment and thought I was talking about the screenshot of my IDE?

Reply:
Ok so the changes are required, doing `request.context = {...}` is deprecated by my changes. The rationals are found in the PR message.

> Maybe you misunderstood my comment and thought I was talking about the screenshot of my IDE?

I was :sweat_smile: